rocprim/include/rocprim/device/device_binary_search.hpp

```diff
@@ -85,6 +85,43 @@ hipError_t binary_search(void * temporary_storage,
 
 } // end of detail namespace
 
+/// \brief Performs a device-level lower bound check.
+///
+/// \par Overview
+/// Runs multiple lower bound checks in parallel (one for each \p needle in \p needles ).
```

Comment:
extra whitespace after needles

Reply:
I've switched this to using the equivalent: "<tt>needles</tt>", since removing the space causes the closing parenthesis to be considered part of the expression the \p formats.